docs/animations/Lottie.md

```diff
@@ -14,7 +14,7 @@ Lottie simplifies the design-to-code workflow for bringing engaging, interactive
 ![Lottie Gif](../resources/images/Animations/Lottie/LottieDocs_Intro.gif)
 
 > [!div class="nextstepaction"]
-> [Try it in the sample app](uwpct://Animations?sample=Lottie)
+> [Try it in the sample app](https://www.microsoft.com/en-us/p/lottie-samples/9n3j5tg8ff7f?rtc=1)
```

Comment:
The link we have works, it just brings you to the current sample app doc page we have for Lottie:

![image](https://user-images.githubusercontent.com/24302614/148814041-154ad514-4c67-4bb9-91fe-a96dd250d9e9.png)

Though it probably does make sense to link to your full Lottie sample app instead. However, we shouldn't include the locale `en-us` in the link. Do you have an aka.ms link for the sample app too? That may make it easier to update if needed in the future without having to change the docs again in the future.